tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -556,17 +556,17 @@ func TestTcpMetrics(t *testing.T) {
 	if err != nil {
 		fatalf(t, "Could not build prometheus API client: %v", err)
 	}
-	query := fmt.Sprintf("istio_tcp_sent_bytes_total{destination_app=\"%s\"}", "mongodb")
+	query := fmt.Sprintf("istio_tcp_sent_bytes_total{destination_app.kubernetes.io/name=\"%s\"}", "mongodb")
 	want := float64(1)
 	validateMetric(t, promAPI, query, "istio_tcp_sent_bytes_total", want)
 
-	query = fmt.Sprintf("istio_tcp_received_bytes_total{destination_app=\"%s\"}", "mongodb")
+	query = fmt.Sprintf("istio_tcp_received_bytes_total{destination_app.kubernetes.io/name=\"%s\"}", "mongodb")
 	validateMetric(t, promAPI, query, "istio_tcp_received_bytes_total", want)
 
-	query = fmt.Sprintf("sum(istio_tcp_connections_opened_total{destination_app=\"%s\"})", "mongodb")
+	query = fmt.Sprintf("sum(istio_tcp_connections_opened_total{destination_app.kubernetes.io/name=\"%s\"})", "mongodb")
```

Comment:
please revert this part of the changes. The labels used in queries should not be changed.

/cc @gargnupur